.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+<!--
+- Please ensure this PR contains a single or a list of atomic commits.
+- At least one commit must have a message (What) and a description (Why).
+- Evey commit should have a message (What) and a description (Why).
+- Evey PR must have a description.
```

Comment:
Typo `s/Evey/Every`

Also I think these rules (L3-5) are overlapping, we don't need to say "At least one" if we then say "Every commit should have..."

I think we can just say `Every commit should have a message/title (What) and a description/body (Why)`

We might also want to say the description should include sufficient context to describe the change being made, and where possible all code changes should be split into a series of commits instead of one large change?

Reply:
>Also I think these rules (L3-5) are overlapping, we don't need to say "At least one" if we then say "Every commit should have..."
I think we can just say Every commit should have a message/title (What) and a description/body (Why)

If the PR is single commit it must have a message and desc. If the PR is multiple commits at least one must have a desc and the others should. In any case the PR must have a desc.

If we think ^^ that is well understood by "Every commit should have a message/title (What) and a description/body (Why)", I'll update

Reply:
Ah I get the PR/commit and should/must distinction now, it wasn't obvious on first reading :)

How about:

* Every PR must have a description.
* Please ensure large code changes are split into a series of logically independent commits
* Every commit should have a message/title (What) and a description/body (Why)

I'm not sure if we need to explicitly call out "At least one" or if "should " is sufficiently open to interpretation, up to you :)